skbio/sequence/_sequence.py

```diff
@@ -1088,8 +1155,10 @@ def __repr__(self):
         ...     'quality': [3, 10, 11, 10],
         ...     'exons': [True, True, False, True]
         ... }
-        >>> Sequence('ACGT', metadata=metadata,
+        >>> seq = Sequence('ACGT', metadata=metadata,
         ...          positional_metadata=positional_metadata)
+        >>> _ = seq.interval_metadata.add([(0, 2)], metadata={'gene': 'sagA'})
```

Comment:
yes. just to avoid doctest fail or adding the `repr` print here. I can use `# noqa` to skip doctest